packages/@react-spectrum/s2/src/Table.tsx

```diff
@@ -11,85 +11,1089 @@
+          <span className={cellContent({...tableVisualOptions, align: align || 'start'})}>{children}</span>
+        </>
       )}
-      {selectionBehavior === 'toggle' && (
-        <Cell>
-          <Checkbox slot="selection" />
+    </RACCell>
+  );
+}
+
+// TODO: reminder, these dark theme colors were grabbed from the spectrum tokens but they feel quite dark IMO. Will comment in design file
+const rowBackgroundColor = {
+  default: 'gray-25',
+  isFocusVisibleWithin: 'gray-900/7', // table-row-hover-color
+  isHovered: 'gray-900/7', // table-row-hover-color
+  isPressed: 'gray-900/10', // table-row-hover-color
+  isSelected: {
+    // TODO: I've tried adding +50 to the dark color opacity and it doesn't look too bad
+    default: lightDark('informative-900/10', 'informative-700/10'), // table-selected-row-background-color, opacity /10
+    isFocusVisibleWithin: lightDark('informative-900/15', 'informative-700/15'), // table-selected-row-background-color, opacity /15
+    isHovered: lightDark('informative-900/15', 'informative-700/15'), // table-selected-row-background-color, opacity /15
+    isPressed: lightDark('informative-900/15', 'informative-700/15') // table-selected-row-background-color, opacity /15
+  },
+  isQuiet: {
+    // TODO: there aren't designs for quiet + selected? For now I've made it the same as non-quiet
+    default: 'transparent',
+    isFocusVisibleWithin: 'gray-900/7', // table-row-hover-color
+    isHovered: 'gray-900/7', // table-row-hover-color
+    isPressed: 'gray-900/10', // table-row-hover-color
+    isSelected: {
+      default: lightDark('informative-900/10', 'informative-700/10'), // table-selected-row-background-color, opacity /10
+      isFocusVisibleWithin: lightDark('informative-900/15', 'informative-700/15'), // table-selected-row-background-color, opacity /15
+      isHovered: lightDark('informative-900/15', 'informative-700/15'), // table-selected-row-background-color, opacity /15
+      isPressed: lightDark('informative-900/15', 'informative-700/15') // table-selected-row-background-color, opacity /15
+    }
+  },
+  forcedColors: {
+    default: 'Background'
+  }
+} as const;
+
+const row = style<RowRenderProps & S2TableProps>({
+  height: 'full',
+  position: 'relative',
+  boxSizing: 'border-box',
+  backgroundColor: rowBackgroundColor,
+  '--rowBackgroundColor': {
+    type: 'backgroundColor',
+    value: rowBackgroundColor
+  },
+  '--rowFocusIndicatorColor': {
+    type: 'backgroundColor',
+    value: {
+      default: 'focus-ring',
+      forcedColors: 'Highlight'
+    }
+  },
+  // TODO: outline here is to emulate v3 forcedColors experience but runs into the same problem where the sticky column covers the outline
+  // This doesn't quite work because it gets cut off by the checkbox cell background masking element, figure out another way. Could shrink the checkbox cell's content even more
+  // and offset it by margin top but that messes up the checkbox centering a bit
```

Comment:
for context, this is what it looks like in HCM for windows for S1 tableview:
<img width="443" alt="image" src="https://github.com/user-attachments/assets/1853f790-c66e-472d-9b4e-db0539bf15c2">